### DISCUSSION TITLE
Play store

### DISCUSSION THREAD

**Discussion by Yeasir0032** [Innovative-CST/Android-Code-Editor#2] [Q&A] [answered]
Does this is available in play store?
If yes, then provides the link

**Comment by DevVigilante** [comment, marked as answer]
It is not available on play store but you can download latest debug build from [here](https://github.com/TS-Code-Editor/Android-Code-Editor/suites/14075307227/artifacts/786818894)

**Reply by Yeasir0032** [reply]
Thank you

**Reply by DevVigilante** [reply]
Whenever app updated you get debug build on [Github action](https://github.com/TS-Code-Editor/Android-Code-Editor/actions)
